internal/runtime/z_face.go

```diff
@@ -322,4 +325,95 @@ func Implements(T, V *abi.Type) bool {
 	return false
 }
 
+func EfaceEqual(v, u eface) bool {
```

Comment:
https://github.com/goplus/llgo/blob/main/internal/abi/type.go#L53

How to use `abi.Type.Equal`?